doc/api/child_process.md

```diff
@@ -704,6 +706,22 @@ see [V8 issue 7381](https://bugs.chromium.org/p/v8/issues/detail?id=7381).
 
 See also: [`child_process.exec()`][] and [`child_process.fork()`][].
 
+### `child_process.escapeArgument(args)`
+
+* `args` {string[]} list of string arguments.
```

Comment:
```suggestion
* `arg` {string} a command line argument to escape.
```

This matches the expectation of the implementation. Additionally the signature above will need to be adjusted accordingly.